games-fps/yamagi-quake2/yamagi-quake2-8.00.ebuild

```
@@ -0,0 +1,151 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop wrapper
+
+CTF_V="1.08"
+ROGUE_V="2.08"
+XATRIX_V="2.09"
+REF_VK_V="1.0.1"
+
+DESCRIPTION="Quake 2 engine focused on single player"
+HOMEPAGE="https://www.yamagi.org/quake2/"
+SRC_URI="https://deponie.yamagi.org/quake2/quake2-${PV}.tar.xz
+	ctf? ( https://deponie.yamagi.org/quake2/quake2-ctf-${CTF_V}.tar.xz )
+	rogue? ( https://deponie.yamagi.org/quake2/quake2-rogue-${ROGUE_V}.tar.xz )
+	xatrix? ( https://deponie.yamagi.org/quake2/quake2-xatrix-${XATRIX_V}.tar.xz )
+	vulkan? ( https://github.com/yquake2/ref_vk/archive/refs/tags/v${REF_VK_V}.tar.gz ->
+		quake2-ref_vk-${REF_VK_V}.tar.gz )
+"
+S="${WORKDIR}/quake2-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="+client ctf dedicated openal +opengl rogue softrender vulkan xatrix"
+REQUIRED_USE="
+	|| ( client dedicated )
+	client? ( || ( opengl softrender vulkan ) )
+"
+
+COMMON_DEPEND="
+	client? (
+		media-libs/libsdl2[video]
+		!openal? ( media-libs/libsdl2[sound] )
+		opengl? (
+			media-libs/libsdl2[opengl]
+			virtual/opengl
+		)
+	)
+"
+DEPEND="${COMMON_DEPEND}
+	client? ( vulkan? ( dev-util/vulkan-headers ) )
+"
+RDEPEND="${COMMON_DEPEND}
+	client? ( openal? ( media-libs/openal ) )
+"
+
+DOCS=( CHANGELOG README.md doc )
+
+mymake() {
+	emake \
+		VERBOSE=1 \
+		WITH_SYSTEMWIDE=yes \
+		WITH_SYSTEMDIR="${EPREFIX}"/usr/share/quake2 \
+		WITH_OPENAL=$(usex openal) \
+		"$@"
+}
+
+src_compile() {
+	local targets=( game )
```

One last thing, noticed this fails to build with gcc-config[-native-symlinks], adding tc-export seems enough (also inherit toolchain-funcs)
```suggestion
src_compile() {
	tc-export CC

	local targets=( game )
```